hosts/hetzci/dev/casc/pipelines/ghaf-hw-test-manual.groovy

```diff
@@ -38,6 +38,21 @@ properties([
         ]
       ]
     ],
+    [
+      $class: 'ChoiceParameter',
+      name: 'TESTAGENT_HOST',
+      choiceType: 'PT_RADIO',
+      description: '''
+        Select wanted testagent-host if Jenkins controller is connected with multiple testagent-hosts.'''.stripIndent(),
```

Comment:
Suggested change:
```
      description: '''
        Select the testagent-host. This parameter allows specifying the exact testagent in case Jenkins controller is
        connected with multiple agents. Can be used together with both IMG_URL and DEVICE_TAG parameters.'''.stripIndent(),
```